packages/vetrina/src/Vetrina/Vetrina.purs

```diff
@@ -85,11 +85,9 @@ data PurchaseState
   = NewPurchase
   | CapturePayment PaymentTerminalUrl
   | ProcessPayment
-  | PurchaseFailed
+  | PurchaseFailed (Maybe OrderFailure)
```

Comment:
This should be `PurchaseFailed OrderFailure`, as we should always know what error happened.

Reply:
What error would you return here?

https://github.com/KSF-Media/affresco/blob/master/packages/vetrina/src/Vetrina/Vetrina.purs#L205

Reply:
Depends on what the `OrderStatusFailReason` is (https://github.com/KSF-Media/affresco/blob/master/packages/components/src/Bottega.purs#L82-L89).

I'm currently implementing the `SubscriptionExistsError` one, but I think in other cases we can default to `ServerError` or `UnexpectedError` even though they're not exactly true (It probably would not be beneficial to reflect all the `OrderStatusFailReason` values to the user interface)